Events/Ch8.event

```diff
@@ -0,0 +1,443 @@
+CAM1 0x1f
+FlashCursor(0x1f,50)
+Text(ChSixHasty)
+
+LABEL 0x2
+LABEL 0x3
+LABEL 0x1
+NoFade
+ENDA
+
+
+
+
+// Manual Movement
+
+// Scripted Fights
+
+// Units
+
+Bads12:
+UNIT 0x50 0x27 0x00 Level(1, Enemy, 1) [23,8] 0x00 0x00 0x0 0x00 [0x40, 0x4E] AttackInRangeAI
+UNIT 0x90 0x4E 0x50 Level(7, Enemy, 1) [19,3] 0x00 0x00 0x0 0x00 [0x8D] AttackInRangeAI
+UNIT 0x90 0x42 0x50 Level(7, Enemy, 1) [19,6] 0x00 0x00 0x0 0x00 [0x5A] AttackInRangeAI
+UNIT 0x90 0x42 0x50 Level(7, Enemy, 1) [22,4] 0x00 0x00 0x0 0x00 [0x5A] AttackInRangeAI
+UNIT 0x90 0x4E 0x50 Level(7, Enemy, 1) [16,4] 0x00 0x00 0x0 0x00 [0x8D] AttackInRangeAI
+UNIT 0x90 0x19 0x50 Level(6, Enemy, 1) [13,5] 0x00 0x00 0x0 0x00 [0x2D] AttackInRangeAI
+UNIT 0x90 0x4E 0x50 Level(6, Enemy, 1) [12,9] 0x0 0x00 0x0 0x00 [0x1C, 0x18] AttackInRangeAI
+UNIT 0x90 0x44 0x50 Level(5, Enemy, 1) [0,7] 0x2 0x00 0x0 0x00 [0x3F, 0x6C] AttackInRangeAI
+UNIT 0x90 0x44 0x50 Level(5, Enemy, 1) [8,3] 0x0 0x00 0x0 0x00 [0x3F] AttackInRangeAI
+UNIT 0x66 0x27 0x50 Level(1, Enemy, 0) [19,1] 0x00 0x00 0x0 0x00 [0x00] GuardTileAI
+UNIT
+
+BadsMore12:
+UNIT 0x90 0x13 0x50 Level(7, Enemy, 1) [23,6] 0x2 0x00 0x0 0x00 [0x0D] AttackInRangeAI
+UNIT 0x90 0x13 0x50 Level(7, Enemy, 1) [21,8] 0x0 0x00 0x0 0x00 [0x0F] PursueWithoutHeedAI
+UNIT 0x90 0x19 0x50 Level(6, Enemy, 1) [15,7] 0x0 0x00 0x0 0x00 [0x30] PursueWithoutHeedAI
+UNIT 0x90 0x3F 0x50 Level(6, Enemy, 1) [14,4] 0x0 0x00 0x0 0x00 [0x26, 0x22] PursueWithoutHeedAI
+UNIT
+
+
+Bads12H:
+UNIT 0x50 0x27 0x00 Level(1, Enemy, 1) [23,8] 0x00 0x00 0x0 0x00 [0x39, 0x40, 0x4E] AttackInRangeAI
+UNIT 0x90 0x4E 0x50 Level(8, Enemy, 1) [19,3] 0x00 0x00 0x0 0x00 [0x8D, 0x1c] AttackInRangeAI
+UNIT 0x90 0x42 0x50 Level(8, Enemy, 1) [19,6] 0x00 0x00 0x0 0x00 [0x5A] AttackInRangeAI
+UNIT 0x90 0x42 0x50 Level(7, Enemy, 1) [22,4] 0x00 0x00 0x0 0x00 [0x5A] AttackInRangeAI
+UNIT 0x90 0x4E 0x50 Level(7, Enemy, 1) [16,4] 0x00 0x00 0x0 0x00 [0x8D] AttackInRangeAI
+UNIT 0x90 0x13 0x50 Level(7, Enemy, 1) [23,7] 0x2 0x00 0x0 0x00 [0x0D] AttackInRangeAI
+UNIT 0x90 0x19 0x50 Level(7, Enemy, 1) [13,5] 0x00 0x00 0x0 0x00 [0x2D] AttackInRangeAI
+UNIT 0x90 0x44 0x50 Level(6, Enemy, 1) [0,7] 0x2 0x00 0x0 0x00 [0x40, 0x6C] AttackInRangeAI
+UNIT 0x90 0x44 0x50 Level(6, Enemy, 1) [8,3] 0x0 0x00 0x0 0x00 [0x3F] AttackInRangeAI
+UNIT 0x66 0x27 0x50 Level(1, Enemy, 0) [19,1] 0x00 0x00 0x0 0x00 [0x00] GuardTileAI
+UNIT
+
+BadsMore12H:
+UNIT 0x90 0x13 0x50 Level(8, Enemy, 1) [23,6] 0x0 0x00 0x0 0x00 [0x0F, 0x04] PursueWithoutHeedAI
+UNIT 0x90 0x19 0x50 Level(8, Enemy, 1) [15,7] 0x0 0x00 0x0 0x00 [0x2e, 0x2e] PursueWithoutHeedAI
+UNIT 0x90 0x3F 0x50 Level(7, Enemy, 1) [14,4] 0x0 0x00 0x0 0x00 [0x26, 0x21] PursueWithoutHeedAI
+UNIT 0x90 0x4E 0x50 Level(7, Enemy, 1) [12,9] 0x0 0x00 0x0 0x00 [0x1C, 0x18] AttackInRangeAI
+UNIT
+
+TheCookieBatterisintheFridge12:
+UNIT 0x0D 0x0A 0x00 Level(6, NPC, 0) [5,8] 0x00 0x00 0x0 0x00 [0x14, 0x96, 0x6f] PursueWithoutHeedAI

```

Comment:
Add definitions for the new characters to custom definitions.event and use them here. It makes the event code more legible and allows for single point of control.